tests/functional/test_contracts.py

```diff
@@ -164,6 +164,32 @@ def test_contract_logs_range(contract_instance, owner, assert_log_values):
     assert_log_values(logs[0], 1)
 
 
+def test_contract_logs_range_by_address(contract_instance, owner, assert_log_values):
```

Comment:
this test is failing and im not sure why! It should be fine but I just don't get any logs (returns empty list)